pilot/cmd/pilot-agent/status/server.go

```diff
@@ -178,6 +177,17 @@ func (s *Server) handleReadyProbe(w http.ResponseWriter, _ *http.Request) {
 	s.mutex.Unlock()
 }
 
+func (s *Server) handleQuit(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "POST" {
+		http.Error(w, "Method Not Allowed", http.StatusMethodNotAllowed)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
+	_, _ = w.Write([]byte("OK"))
+	log.Infof("handling %s, and notify pilot-agent to exit", quitPath)
+	notifyExit()
+}
```

Comment:
will a SIGTERM trigger this code? https://github.com/istio/istio/blob/13cb3422ae47f95dfbc829a327724a56929cab3b/pilot/pkg/proxy/agent.go#L264-L272

Reply:
yes， this is the target